symfony/plugins/orangehrmAttendancePlugin/config/routes.yaml

```diff
@@ -128,3 +133,13 @@ apiv2_attendance_edit_attendance_record:
   path: /api/v2/attendance/employees/record/{id?}
   controller: OrangeHRM\Attendance\Controller\AttendanceMockController::sendAttendanceEditResponse
   methods: [ GET, PUT ]
+
+apiv2_attendance_summary_reports:
+  path: /api/v2/attendance/reports
+  controller: OrangeHRM\Attendance\Controller\ReportMockController::getReportHeaders
+  methods: [ GET ]
+
+apiv2_attendance_summary_reports_data:
+  path: /api/v2/attendance/reports/data
+  controller: OrangeHRM\Attendance\Controller\ReportMockController::getReportData
+  methods: [ GET ]
```

Comment:
@vageeshorangehrm we have API for this? `attendance_summary_report`?

Reply:
yes rajitha, we have the BE API for attendance report.

Reply:
connected BE API